.github/workflows/nightly.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           key: ${{ runner.os }}-microbenchmarks
 
   benchmarks:
-    runs-on: buildjet-8vcpu-ubuntu-2204
+    runs-on: warp-custom-slatedb-x64-32vCPU-128GB-RAM-300GB-DISK
```

Comment:
where's this config specified?

Reply:
This is configured in the WarpBuild UI. I don't know yet if there's a team setting. Still trying to figure out our options. Several folks have recommended Ubicloud and Depot.dev instead. Going to do some digging tomorrow.

All of the providers seem to do this. They use the "runs-on" to flag their runners, and then they take over and execute the workflow.

Reply:
Hi @criccomini, this is Enes from Ubicloud, ready to answer any questions you may have. 

While we don't offer BYOC, our prices are 10 times more affordable than official GitHub's runners.

I have tested your workflows on different sized Ubicloud runners.

https://github.com/enescakir/slatedb/actions/runs/12492978671

![CleanShot 2024-12-25 at 18 49 12@2x](https://github.com/user-attachments/assets/9ca96368-f5d3-4471-a20c-12054de363e3)